cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('template spec', () => {
+  it('passes', () => {
+    // go to the Skosmos front page
+    cy.visit('http://localhost/Skosmos')
+    // click on the first vocabulary in the list
+    cy.get('#vocabulary-list').find('a').first().click()
+    // check that we are still within the Skosmos URL space
+    cy.url().should('include', 'http://localhost/Skosmos/')
+    // check that we are on the vocab home page
+    cy.get('#vocab-info')
```

Comment:
This should probably still do something when it gets the element. I can't recall if it will return an error if the element is not found, but I suspect it might instead just return `null` or `undefined`, without raising an error.